src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -430,7 +437,18 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
               break
             case KeyManager.KeepKey:
               try {
-                const localKeepKeyWallet = state.keyring.get(localWalletDeviceId)
+                let localKeepKeyWallet: HDWallet | null | undefined =
```

Comment:
use const with IIFE over let/mutate, no need to explicitly specify type here as the return from the keyring is inferred